test/e2e_node/runtime_conformance_test.go

```diff
@@ -132,7 +132,7 @@ while true; do sleep 1; done
 					By("it should get the expected 'State'")
 					Expect(GetContainerState(status.State)).To(Equal(testCase.State))
 
-					By("it should be possible to delete [Conformance][NodeConformance]")
+					By("it should be possible to delete [NodeConformance]")
```

Comment:
It's weird that this is a subpart of this test case, or sequence of test cases. I'm not sure it's right even for NodeConformance.